testgrid/config.yaml

```diff
@@ -5706,11 +5712,20 @@ dashboards:
   - name: autoscaling-vpa-full
     test_group_name: ci-kubernetes-e2e-autoscaling-vpa-full
 
+- name: sig-aws-alb-ingress-controller
+  dashboard_tab:
+  - name: lint
+    test_group_name: pull-aws-alb-ingress-controller-lint
+    description: "aws alb ingress controller code lint"
+  - name: test
+    test_group_name: pull-aws-alb-ingress-controller-test
+    description: "aws alb ingress controller tests"
```

Comment:
Just a minor comment. I assume this is unit test? Could you mention this in the description to make it explicit?

Reply:
the purpose is to use the same "test" target to run all tests. (otherwise e2e test will be running before unit test succeeds. which it should't)